app/views/root/index.erb

```diff
@@ -1 +1,21 @@
-<H1> This is root page </H1>
\ No newline at end of file
+<% if user_signed_in? %>
+  <%= stylesheet_link_tag 'article' %>
+  <% @articles.each do |article| %>
+    <div class="article">
+      <div class="article-name-container">
+        <div>
+          <%= article.user.name %>
+        </div>
+        <div>
+          <%= article.user.surname %>
+        </div>
+      </div>
+      <div>
+        <%= article.message %>
+      </div>
+      <div>Posted <%= time_ago_in_words(article.created_at) %> ago</div>
+    </div>
+  <% end %>
+<% else %>
+  <h1> You should <%= link_to "sign in", new_session_path(:user) %> to see articles </h1>
```

Comment:
I don't think it is a title of the page, so I'd rather give it a `p` tag and add some styling instead